Sources/OpenCombine/Published.swift

```diff
@@ -145,14 +153,14 @@ public struct Published<Value> {
     ///
     /// - Parameter initialValue: The publisher's initial value.
     public init(wrappedValue: Value) {
-        storage = .value(wrappedValue)
+        _storage = Box(wrappedValue: .value(wrappedValue))
     }
 
     /// The property for which this instance exposes a publisher.
     ///
     /// The `projectedValue` is the property accessed with the `$` operator.
     public var projectedValue: Publisher {
-        mutating get {
+        get {
```

Comment:
In Apple's Combine, this getter is `mutating`, even though it doesn't actually mutate anything. Let's be compatible.

Reply:
This breaks the build as it requires use to remove `nonmutating` from `set` on `internal var objectWillChange`:

```
 Published.swift:131

   NOTE   cannot use mutating getter on immutable value: 'self' is immutable
  131 |             projectedValue.subject.objectWillChange = newValue
      |              ^
```

@kateinoigakukun WDYT? Are there other options to work around the error?